interface.html

```diff
@@ -52,6 +55,38 @@
             <span class="text-helper">Users will use the password on this field to log in.</span>
           </div>
         </div>
+        <h3></h3>
+        <div class="form-group clearfix">
+          <div class="col-sm-4 control-label">
+            <label for="loggedInUserTime">Keep the user logged in for
```

Comment:
The `for` attribute for `<label>` should reference the `id` attribute for the form element it's bound to. See https://www.w3schools.com/tags/tag_label.asp

```suggestion
            <label for="logged-in-user-time">Keep the user logged in for
```

Reply:
did the changes.